modules/core/react/lib/theming/README.md

```diff
@@ -0,0 +1,23 @@
+### Custom Theme
```

Comment:
Need to build out this readme and link to it from the core readme.

Reply:
Add note about adding custom theming for a specific component (green `Switch` for example)